packages/content/src/slugs.ts

```diff
@@ -2,7 +2,7 @@ import { isGuid } from "@esri/hub-common";
 
 /**
  * Parse item ID and layer ID (if any) from dataset record ID
- *
+ * **DEPRECATED: import from hub-common instead**
```

Comment:
we want to remove the implementations in this file, so it should be just a deprecation message and then:

```ts
export { parseDatasetId, isSlug, addContextToSlug, removeContextFromSlug } from `@esri/hub-common`;
```

We'll then want to remove slugs.test.ts too.